pkg/cmd/cli/kubeflowmodelregistry/kfmr.go

```diff
@@ -429,6 +435,16 @@ func (m *ModelPopulator) GetUsage() *string {
 	return m.getStringPropVal(brdgtypes.UsageKey)
 }
 
+func (m *ModelPopulator) GetTechDocs() *string {
```

Comment:
I remembered after signing off Friday that the unit test link I provided for the adding of the license field could also be applied here; especially given the defaulting applied.